docs/index.md

```diff
@@ -2,10 +2,25 @@
 
 ---
 
-The documentation on this site is written for developers.
-People who want to contribute to Decred, or create their own Decred based applications.
+If you're looking to develop Decred-based applications, become a 
+contributor, or simply gain a deeper understanding of Decred, 
```

Comment:
Theres a lot of commas in this opening paragraph.

Who are you, and what have you done with the real @s-ben? :joy:

Reply:
Blame @RichardRed0x. His sick deviant comma habit is a contagious word virus.